src/.npmignore

```diff
@@ -0,0 +1,2 @@
+*.test.*
+
```

Comment:
Tested this with `npm pack`:

Before: 128 files being packaged.
After: 124 files being packaged.

There are exactly 4 test files inside `src`, so this seems correct.